ext/dom/html5_serializer.c

```diff
@@ -335,8 +323,7 @@ static zend_result dom_html5_serialize_node(dom_html5_serialize_context *ctx, co
 	return SUCCESS;
 }
 
-/* https://html.spec.whatwg.org/multipage/parsing.html#serialising-html-fragments (Date 2023-10-18)
- * Note: this serializes the _children_, excluding the node itself! */
+/* Note: this serializes the _children_, excluding the node itself! */
```

Comment:
Normal that the spec link is removed?

Reply:
This link is now at the top of the file.

Reply:
The diff view being confusing